addons/l10n_latam_check/models/account_payment.py

```diff
@@ -1,328 +1,252 @@
+    l10n_latam_move_check_ids = fields.Many2many(
+        comodel_name='l10n_latam.check', relation='account_payment_account_payment_check_rel',
+        required=True,
         copy=False,
-        check_company=True,
-    )
-    l10n_latam_check_operation_ids = fields.One2many(
-        comodel_name='account.payment',
-        inverse_name='l10n_latam_check_id',
-        string='Check Operations',
-        readonly=True,
-    )
-    l10n_latam_check_current_journal_id = fields.Many2one(
-        comodel_name='account.journal',
-        string="Check Current Journal",
-        compute='_compute_l10n_latam_check_current_journal', store=True,
+        string="Checks Operations"
     )
     # Warning message in case of unlogical third party check operations
-    l10n_latam_check_warning_msg = fields.Text(
-        compute='_compute_l10n_latam_check_warning_msg',
-    )
-    l10n_latam_check_number = fields.Char(
-        compute='_compute_l10n_latam_check_number', inverse='_inverse_l10n_latam_check_number',
-    )
-    # New third party check info
-    l10n_latam_check_bank_id = fields.Many2one(
-        comodel_name='res.bank',
-        string='Check Bank',
-        compute='_compute_l10n_latam_check_bank_id', store=True, readonly=False,
-    )
-    l10n_latam_check_issuer_vat = fields.Char(
-        string='Check Issuer VAT',
-        compute='_compute_l10n_latam_check_issuer_vat', store=True, readonly=False,
-    )
-    l10n_latam_check_payment_date = fields.Date(
-        string='Check Cash-In Date',
-        help="Date from when you can cash in the check, turn the check into cash",
-        readonly=False,
-    )
-
-    # This is a technical field for the view only
-    l10n_latam_manual_checks = fields.Boolean(
-        related='journal_id.l10n_latam_manual_checks',
-    )
+    l10n_latam_check_warning_msg = fields.Text(compute='_compute_l10n_latam_check_warning_msg')
+    amount = fields.Monetary(compute="_compute_amount", readonly=False, store=True)
 
-    @api.depends('check_number')
-    def _compute_l10n_latam_check_number(self):
-        """ This dummy computed field is added for two reasons:
-        1. add a new field so that we don't need to modify attrs on the views for the original check_number field
-        (not nice on terms of inheritance)
-        2. if we set it as related (with readonly=False) it didn't work properly for our use case: If the user changes
-        the proposed number the value was not saved. This computed with inverse does the trick"""
+    @api.depends('l10n_latam_move_check_ids.amount', 'l10n_latam_new_check_ids.amount', 'payment_method_code')
+    def _compute_amount(self):
         for rec in self:
-            rec.l10n_latam_check_number = rec.check_number
+            checks = rec.l10n_latam_new_check_ids if rec._is_latam_check_payment(check_subtype='new_check') else rec.l10n_latam_move_check_ids
+            if checks:
+                rec.amount = sum(checks.mapped('amount'))
```

Comment:
What if you have twice the same amount?

Reply:
No problem, I beleive it only returns unique on recordsets
Actually this use case is shown on documentation: https://www.odoo.com/documentation/17.0/es/developer/reference/backend/orm.html#field-access
And also it's found a lot on odoo 
![image](https://github.com/odoo/odoo/assets/3016656/8e17ffb9-0b1a-435a-96d0-60d664e1007c)

Reply:
Done